supabase/functions/oauth/access-token.ts

```diff
@@ -5,6 +5,7 @@ import jsonpointer from "https://esm.sh/jsonpointer.js";
 import { returnPostgresError, handlebarsHelpers } from "../_shared/helpers.ts";
 import { corsHeaders } from "../_shared/cors.ts";
 import { supabaseClient } from "../_shared/supabaseClient.ts";
+import _ from "https://cdn.skypack.dev/lodash";
```

Comment:
Is this import needed?

Reply:
Good point, leftover, at first I was using `_.isEmpty`